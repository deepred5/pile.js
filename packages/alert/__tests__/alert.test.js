/**
 * @author renmaomin@126.com
 */
import React from 'react';
import { shallow } from 'enzyme';
import Alert from '../src';

describe('Alert suite', () => {
  it('render a alert', () => {
    const wrapper = shallow(
      <Alert show btnText="确定" callBack={() => {}}>
        内容
      </Alert>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
