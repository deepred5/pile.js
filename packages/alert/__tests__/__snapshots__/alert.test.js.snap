// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert suite render a alert 1`] = `
<CSSTransition
  classNames="pile-alert-animate"
  in={false}
  onEnter={[Function]}
  onExited={[Function]}
  timeout={200}
  unmountOnExit={true}
>
  <div
    className="pile-alert"
  >
    <div
      className="pile-alert-mask"
    />
    <div
      className="pile-alert-box"
    >
      <i
        className="pile-icon-check pile-alert-icon"
        style={
          Object {
            "display": "none",
          }
        }
      />
      <div
        className="pile-alert-title"
      >
        提示
      </div>
      <div
        className="pile-alert-content"
      >
        内容
      </div>
      <div
        className="d-btns pile-btn-alert"
        onClick={[Function]}
        onKeyPress={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="btn-orange"
        >
          确定
        </span>
      </div>
    </div>
  </div>
</CSSTransition>
`;
